packages/app/src/containers/ExportInspector.tsx

```diff
@@ -17,15 +17,20 @@ import {
 } from '../contexts/ApplicationStateContext';
 import useCanvasKit from '../hooks/useCanvasKit';
 import { renderImageFromCanvas } from '../utils/renderImageFromCanvas';
+import JSZip from 'jszip';
 
-async function saveFile(name: string, data: Uint8Array) {
+async function saveFile(
+  name: string,
+  fileFormat: string,
+  data: Uint8Array | ArrayBuffer,
+) {
   const file = new File([data], name, {
-    type: 'image/png',
+    type: `${fileFormat === 'zip' ? 'application' : 'image'}/${fileFormat}`,
```

Comment:
We'll leave this for now, but I'm going to add some file type stuff to utils and will change this then